[+] Add English translation of homoyamakaze's entry. (#82)
one-among-us/data: people/homoyamakaze/page.en.md

---
name: homoyamakaze
info:

>     alias: 浅羽猫头,穗也猫猫头,qianyumaotou,QianYuMaoTou,Qian Yu Mao Tou,Homoyamakaze

Reviewer: ```suggestion
    alias: Qian Yu Mao Tou, Sui Ye Mao Mao Tou, Homoyamakaze
```

    location: Lanzhou, Gansu, China
---

## Description

[Qian Yu Mao Tou](https://twitter.com/homoyamakaze)
wrote the first article about [how to get a diagnosis of gender dysphoria in Liaoning](https://mtf.wiki/zh-cn/docs/psyco/liaoning/liu-hong/)
on [MtF.wiki](https://mtf.wiki).
She also helped many trans people in person to get their diagnosis and prescriptions.

> She was able to help many people to get affordable HRT medicine while working together with Qian Yu Zhi Xue, number of boxes exceeding a hundred.

Reviewer: ```suggestion
Working together with Qian Yu Zhi Xue, she helped many people get affordable HRT medicine, amount of which was over a hundred boxes.
```

<PhotoScroll photos={[ '${path}/photos/1.jpg', ]} />  
<PhotoScroll photos={[ '${path}/photos/2.png', ]} />

In the evening of 2022 November 9th, she took her life in her house because of a bipolar disorder episode resulted from online violence.

> The incident was recorded in legal documents by her girlfriend [Hui Ye You Zi](https://twitter.com/YuzuTvT).

Reviewer: ```suggestion
The incident was recorded in legal documents by her girlfriend [Sui Ye You Zi](https://twitter.com/YuzuTvT).
```

<PhotoScroll photos={[ '${path}/photos/3.jpg', ]} />  
<PhotoScroll photos={[ '${path}/photos/4.jpg', ]} />

> Confirmed on twitter by both [Hui Ye You Zi](https://twitter.com/YuzuTvT) and [Lee](https://twitter.com/rbqwansui).

Reviewer: ```suggestion
Confirmed on twitter by both [Sui Ye You Zi](https://twitter.com/YuzuTvT) and [Lee](https://twitter.com/rbqwansui).
```

Contributor of entry:

> [Hui Ye You Zi](https://twitter.com/YuzuTvT)

Reviewer: ```suggestion
[Sui Ye You Zi](https://twitter.com/YuzuTvT)
```
